fluidBuild.config.cjs

```diff
@@ -51,7 +51,9 @@ module.exports = {
 		"prettier": [],
 		"webpack": ["^tsc", "^build:esnext"],
 		"webpack:profile": ["^tsc", "^build:esnext"],
-		"clean": [],
+		"clean": {
+			before: ["*"],
```

Comment:
nit: Might be worth commenting here in the config what the intent is.

Reply:
I will add it in the next PR.